Demo/src/main/res/navigation/nav_graph.xml

```diff
@@ -152,5 +155,10 @@
             app:popUpTo="@id/mainFragment"
             app:popUpToInclusive="false" />
     </fragment>
+    <fragment
+        android:id="@+id/payPalNativeCheckoutFragment"
+        android:name="com.braintreepayments.demo.PayPalNativeCheckoutFragment"
+        android:label="fragment_pay_pal_native_checkout"
+        tools:layout="@layout/fragment_pay_pal_native_checkout" />
```

Comment:
`pay_pal` > `payPal` or `paypal`. Looks like we use both, not sure which is preferred on Android.

Reply:
I believe if we distinguish PayPal as two different words, then "pay_pal" would be okay, if not "paypal" would be better. I am not a big fan of camel case on xml